src/canisters/governance/request.converters.ts

```diff
@@ -760,3 +761,24 @@ export const fromMakeProposalRequest = (
     neuron_id_or_subaccount: [{ NeuronId: { id: request.neuronId } }],
   };
 };
+
+export const toRegisterVoteRequest = ({
```

Comment:
Maybe worth a dedicted test even if already indirectly tested?

Reply:
Well, I would consider that the types itself are test enough. There is only one important thing happening here and it's the change of type. I consider that the type checking at compilation it's test enough.